android/build.gradle

```diff
@@ -13,20 +13,26 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 16)
+        minSdkVersion safeExtGet('minSdkVersion', 21)
         targetSdkVersion safeExtGet('targetSdkVersion', 26)
         versionCode 1
         versionName "1.0"
         ndk {
             abiFilters "armeabi-v7a", "x86"
         }
     }
+
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
     lintOptions {
        warning 'InvalidPackage'
     }
 }
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'io.sentry:sentry-android:1.7.29'
+    implementation 'io.sentry:sentry-android:2+'
```

Comment:
Is this temp? For GA I'd suggest we pin the version to avoid having to troubleshoot subtle incompatibilties.
It's more work having to bump, test and publish but at least we'd be aware of which combinations are in the wild.

Reply:
agreed, fixed versions are the recommended way